llumnix/arg_utils.py

```diff
@@ -288,20 +292,23 @@ def add_cli_args(
 
         parser.add_argument('--migration-backend',
                             type=str,
-                            choices=['gloo','nccl','rpc'],
+                            choices=['gloo', 'rpc'],
```

Comment:
nccl is removed from choices, but is still asserted, which is incorrect.

Reply:
Get.
Keep NCCL in the choices but remove it from Augments.md.